frontend/src/views/Org/MembersPage/components/OrgGroupsTab/components/OrgGroupsSection/OrgGroupsTable.tsx

```diff
@@ -1,9 +1,10 @@
-import { useMemo, useState } from "react";
```

Comment:
hmmm could we update the title to be just "Groups"? 

<img width="1267" alt="image" src="https://github.com/user-attachments/assets/db2454dd-e20f-44f8-9078-c769180acdc6">

Reply:
done!